synth.py

```diff
@@ -5,26 +5,31 @@
 
 logging.basicConfig(level=logging.DEBUG)
 
-gapic = gcp.GAPICGenerator()
+gapic = gcp.GAPICMicrogenerator()
 
 # tasks has two product names, and a poorly named artman yaml
 versions = ['v1beta1', 'v1']
-
+name = 'redis'
 for version in versions:
-    library = gapic.node_library('redis', version)
+    library = gapic.typescript_library(
+            name, 
+            generator_args={
+                "package-name": f"@google-cloud/{name}"
+            },
+            proto_path=f'google/cloud/{name}/{version}',
+            extra_proto_files=["google/cloud/common_resources.proto"],
+            version=version,
+    )
     s.copy(
         library,
-        excludes=['package.json', 'README.md', 'src/index.js'])
+        excludes=['package.json', 'README.md', 'src/index.ts'])
```

Comment:
were there manual changes in `src/index.ts`? if not we should avoid excluding it.

Reply:
Yes, we are merging two versions there. We don't do it automatically yet.